packages/ui-components/src/components/Icon/paths.tsx

```diff
@@ -1,5 +1,9 @@
 interface IPath {
-  [key: string]: { type: 'stroke' | 'fill'; path: React.ReactNode };
+  [key: string]: {
+    type: 'stroke' | 'fill';
+    path: React.ReactNode;
+    isColor?: boolean; // mean the icon has pre-defined color
+  };
 }
 
 export const paths: IPath = {
```

Comment:
If we flow this way, please make sure the `viewBox` props of the SVG tag are the same